spec/order_spec.rb

```diff
@@ -0,0 +1,65 @@
+require 'order'
+
+describe Order do
+
+  let(:dish1){double(:dish1, name: "Dishy1", price: 3.50)}
+  let(:dish2){double(:dish2, name: "Dishy2", price: 5.00)}
+  let(:dish3){double(:dish3, name: "Dishy3", price: 4.99)}
+  let(:menu1){double(:menu1, select: dish1)}
+  let(:menu2){double(:menu2, select: dish2)}
```

Comment:
Space missing to the left of {.<br>Space missing inside {.<br>Space missing inside }.